packages/storybook/examples/core/HeaderRow.stories.js

```diff
@@ -7,7 +7,12 @@ import TextEllipsis from '@ichef/gypcrete/src/TextEllipsis';
 
 import DebugBox from 'utils/DebugBox';
 
-function BasicUsage() {
+export default {
+    title: '@ichef/gypcrete|HeaderRow',
+    component: HeaderRow,
```

Comment:
Consider add `<HeaderArea>` to props table?
<img width="499" alt="截圖 2020-02-05 下午4 28 12" src="https://user-images.githubusercontent.com/365035/73824090-84b25380-4834-11ea-8f0c-677d1f58c2f4.png">


```suggestion
    component: HeaderRow,
    subcomponents: { HeaderArea },
```

Reply:
好像是一種辦法XD
我順道把 pt.1 的改一改好了，讓 import 進來的 propTypes 都透過 subcomponents 讀得到 (除了 HOC)